scripts/drone/pipelines/integration_tests.star

```diff
@@ -38,11 +40,23 @@ def integration_tests(trigger, prefix):
     services = integration_test_services(edition = "oss")
     volumes = integration_test_services_volumes()
 
-    init_steps = [
+    init_steps = []
+
+    verify_step = verify_gen_cue_step()
+
+    if ver_mode == "pr":
+        # In pull requests, attempt to clone grafana enterprise.
+        init_steps.append(enterprise_setup_step(location = "../grafana-enterpise"))
+
+        # Ensure that verif_gen_cue happens after we clone enterprise
+        # At the time of writing this, very_gen_cue is depended on by the wire step which is what everything else depends on.
+        verify_step["depends_on"].append("clone-enterprise")
```

Comment:
I don't fully understand these steps yet but is it OK for `verify-gen-jsonnet` to not depend on `clone-enterprise`?

Reply:
Good point, we're probably missing out on some possible test cases here. I think it should depend on that since it's earlier in the pipeline. I'll submit that change today.